minecraft/getting-started.md

```diff
@@ -0,0 +1,9 @@
+---
+tag: "getting-started"
```

Comment:
Since this tag is hardly spigot/paper related I would suggest to reflect this in the tags title.